components/tabs/src/atoms/Tab/index.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { link, spacing, typography } from '@govuk-react/lib';
+
+import { BLACK, BORDER_COLOUR, GREY_4, WHITE } from 'govuk-colours';
+import { MEDIA_QUERIES } from '@govuk-react/constants';
+
+const spacingSimple1 = spacing.simple(1);
+const spacingSimple2 = spacing.simple(2);
+const spacingSimple3 = spacing.simple(3);
+const spacingSimple4 = spacing.simple(4);
+const spacingSimple5 = spacing.simple(5);
+
+const StyledListItem = styled('li')({
+  marginLeft: spacingSimple5,
+  ':before': {
+    content: "'— '",
+    marginLeft: -spacingSimple5,
+    paddingRight: spacingSimple1,
+  },
+  [MEDIA_QUERIES.TABLET]: {
+    marginLeft: 0,
+    ':before': {
+      content: 'none',
+    },
+  },
+});
+
+const StyledHyperLink = styled('a')(
+  typography.font({ size: 19 }),
+  link.common(),
+  link.styleDefault,
+  {
+    display: 'inline-block',
+    paddingTop: spacingSimple2,
+    paddingBottom: spacingSimple2,
+  },
+  ({ isActive }) => ({
```

Comment:
where a govuk-frontend BEM class is `govuk-tabs__tab--selected` we should aim to:

- have a component 'tabs' ✅
- component 'tabs' should have an atom 'tab' ✅
- atom tab should have prop 'selected' ❌